presto-main/src/test/java/io/prestosql/sql/query/TestShowQueries.java

```diff
@@ -91,4 +91,23 @@ public void testListingEmptyCatalogs()
             assertions.getQueryRunner().getAccessControl().reset();
         });
     }
+
+    @Test
+    public void testShowColumnsLike()
+    {
+        assertions.assertQuery("SHOW COLUMNS FROM system.runtime.nodes LIKE 'node%'",
+                "VALUES " +
+                        "(CAST('node_id' AS VARCHAR), CAST('varchar' AS VARCHAR) , CAST('' AS VARCHAR), CAST('' AS VARCHAR))," +
+                        "(CAST('node_version' AS VARCHAR), CAST('varchar' AS VARCHAR) , CAST('' AS VARCHAR), CAST('' AS VARCHAR))");
+    }
+
+    @Test
+    public void testShowColumnsWithLikeWithEscape()
+    {
+        assertions.assertFails("SHOW COLUMNS FROM system.runtime.nodes LIKE 't$_%' ESCAPE ''", "Escape string must be a single character");
+        assertions.assertFails("SHOW COLUMNS FROM system.runtime.nodes LIKE 't$_%' ESCAPE '$$'", "Escape string must be a single character");
+        assertions.assertQuery(
+                "SHOW COLUMNS FROM system.runtime.nodes LIKE '%ode$_id' ESCAPE '$'",
+                "VALUES (CAST('node_id' AS VARCHAR), CAST('varchar' AS VARCHAR), CAST('' AS VARCHAR), CAST('' AS VARCHAR))");
```

Comment:
This row would match even without `ESCAPE`. It would be nice to have a test that would actually verify that `ESCAPE` works. 

Like having columns: `a_` and `aa` then `LIKE 'a$_' ESCAPE '$'` would only match `a_`. Can you write such test?